packages/player/package.json

```diff
@@ -5,15 +5,16 @@
   "type": "module",
   "exports": {
     "import": "./dist/index.js",
-    "default": "./dist/index.js"
+    "default": "./dist/index.js",
+    "legacy": "./dist/legacy/index.js"
```

Comment:
Should we add the "types" under "exports" here as well? (to be consistent with the other packages)